rero_ils/modules/stats/serializers.py

```diff
@@ -75,6 +137,38 @@ def _format_csv(self, records):
                 value = StatCSVSerializer.sort_dict_by_key(value)[1]
                 writer.writerow(value)
                 yield line.read()
+        elif record['metadata'].get('type') == StatType.REPORT:
+            # statistics of type report
+            indicator = record['metadata']['config']['category']['indicator']
+            distributions = indicator.get('distributions')
```

Comment:
use `distributions = indicator.get('distributions', [])` and later you could remove ... (see next comment)